src/Http/Request.php

```diff
@@ -92,7 +92,7 @@ public function getEvent(string $url, array $params = []): ResponseInterface
 
         $args = [
             'skipQueryParams' => $params['skipQueryParams'] ?? true,
-            'token' => !empty($params['token']) ? $params['token'] : '',
+            'token' => !empty($params['token']) ? $params['token'] : [],
```

Comment:
isn't params['token'] a string entity?

Reply:
No, is an array with clientID, clientSecret, refreshToken etc, that we use to refresh the token on src/Http/Request->getHeadersWithToken();